content/articles/why-should-healthcare-industries-adopt-cloud-computing/index.md

```diff
@@ -0,0 +1,145 @@
+
+##### Possible solution
+The healthcare facility should be aware that some cloud providers do not adhere to a high degree of compliance. Additionally, a cloud vendor can be compliant but rely on tools from external/secondary vendors to secure the system. However, the secondary vendor may not be compliant, thereby compromising the overall compliance of the cloud solution.
+
+It means that healthcare organizations should carefully evaluate their cloud vendors to ensure that the solutions they are offering meet mandatory regulations and that all documents needed are signed.
+
+#### Risk 4: Organizational unpreparedness
+Before healthcare organizations migrate to the cloud solution, they should adequately prepare. They need to check if their existing systems and processes are well documented and possess enough knowledge on how their systems work. This reduces the chances of errors been transferred to the cloud and gives the organization a chance to fix its current system.
+
+In addition, some clients never trust their cloud solution providers, hence misusing time and resources maintaining their private environment to protect the data in the cloud.
+
+##### Possible solution
+Healthcare organizations are required to prepare adequately for a cloud solution and adopt it. They need to do technological assessments, analyze their existing platforms, prioritize what to move, and finally get the data ready to be migrated.
+
+In addition, healthcare organizations need to familiarize themselves with their selected cloud platform in advance and involve all the users to make them ready for change and accept new ways after migration.
+
+#### Risk 5: Security threats
+It is possible to argue that the cloud is more secure compared to on-premises storage. However, the cloud still faces risks of data breaches and malware attacks. In most cases, it might be the problem with cloud vendors, but employee errors and internal intentional attacks cause some issues.
+
+##### Possible solution
+The healthcare organization must be ready to choose cloud vendors that offer a consistent offline backup solution to fight cloud computing security risks. The organization can recover its data from offline backup rather than paying ransom to the attacker.
+
+Robust encryptions methods also need to be implemented so that information will still be unreadable whenever data breaches.
+
+Healthcare organization needs to offer cybersecurity courses to their staffs. Also, a cyber-security-based assessment should be performed before choosing the cloud vendor, contractor, or any other external collaborators.
+
+### Conclusion
+Migrating to the cloud is a challenging process, and healthcare organizations should define their goals, cloud options, among others. This would enable them to set their goals and determine which system and data components they want to migrate.
```

Comment:
```suggestion
Migrating to the cloud is a challenging process, and healthcare organizations should define their cloud options. This would enable them to set their goals and determine which system and data components they want to migrate.